Source/WebCore/platform/cocoa/ContentFilterUnblockHandlerCocoa.mm

```diff
@@ -179,7 +179,7 @@
 {
 #if HAVE(WEBCONTENTRESTRICTIONS)
     if (m_evaluatedURL) {
-        WebCore::ParentalControlsURLFilter::singleton().allowURL(*m_evaluatedURL, [decisionHandler = WTFMove(decisionHandler)](bool didAllow) {
+        WebCore::ParentalControlsURLFilter::filterWithConfigurationPath({ }).allowURL(*m_evaluatedURL, [decisionHandler = WTFMove(decisionHandler)](bool didAllow) {
```

Comment:
This function can be invoked in UI process (`WebFrameProxy::didHandleContentFilterUnblockNavigation`), so I think you need to pass the path here, or this will update the default file and the change will not be observed by network process.